#Generated From sr_control_gui, Hand Calibration plugin.

sr_calibrations: [
['MFJ4', [[443.06999999999999, -25.0], [1712.74, 0.0], [1981.6800000000001, 25.0]]],
['MFJ3', [[1512.6900000000001, 0.0], [1937.99, 22.5], [1927.8499999999999, 22.5], [2326.8299999999999, 45.0], [2766.0500000000002, 67.5], [3061.0700000000002, 90.0]]],
['MFJ2', [[2521.8899999999999, 0.0], [2169.1399999999999, 22.5], [1672.1700000000001, 45.0], [1226.1800000000001, 67.5], [988.88, 90.0]]],
['MFJ1', [[3348.0100000000002, 0.0], [2883.7199999999998, 22.5], [2075.8099999999999, 45.0], [1289.3399999999999, 67.5], [660.15999999999997, 90.0]]],
['RFJ3', [[1423.2, 0.0], [1786.54, 22.5], [2283.0700000000002, 45.0], [2763.8800000000001, 67.5], [3189.3000000000002, 90.0]]],
['THJ4', [[1440.24, 0.0], [1791.7, 22.5], [2158.1900000000001, 45.0], [2497.8000000000002, 67.5]]],
['THJ3', [[1844.6700000000001, -15.0], [2105.48, 0.0], [2568.7399999999998, 15.0]]],
['RFJ2', [[3039.75, 0.0], [2609.21, 22.5], [2171.3499999999999, 45.0], [1682.3099999999999, 67.5], [1262.4400000000001, 90.0]]],
['LFJ4', [[1975.8499999999999, -25.0], [2152.8099999999999, 0.0], [3453.3600000000001, 25.0]]],
['RFJ1', [[3359.71, 0.0], [2808.9899999999998, 22.5], [2043.3599999999999, 45.0], [1368.6900000000001, 67.5], [801.74000000000001, 90.0]]],
['RFJ4', [[2033.4400000000001, -25.0], [2232.5500000000002, 0.0], [3134.8499999999999, 25.0]]],
['FFJ4', [[3229.04, 0.0], [2215.27, 0.0], [2044.23, 0.0]]],
['FFJ2', [[2749.75, 0.0], [2468.98, 22.5], [1992.28, 45.0], [1639.5899999999999, 67.5], [1333.04, 90.0]]],
['FFJ3', [[2532.79, 0.0], [2006.3, 22.5], [1637.8900000000001, 45.0], [1234.1099999999999, 67.5], [957.25, 90.0]]],
['FFJ1', [[133.12, 0.0], [479.22000000000003, 22.5], [1618.0, 45.0], [2526.0100000000002, 67.5], [3329.3600000000001, 90.0]]],
['WRJ1', [[329.43000000000001, 0.0], [798.17499999999995, 0.0], [1404.5450000000001, 0.0], [1695.6500000000001, 0.0], [1791.79, 0.0]]],
['WRJ2', [[2358.79, 0.0], [1637.1600000000001, 0.0], [1970.1900000000001, 0.0]]],
['THJ5', [[707.13999999999999, -60.0], [1327.0599999999999, -30.0], [1906.78, 0.0], [2849.8899999999999, 30.0], [3338.3600000000001, 60.0]]],
['LFJ2', [[3081.7800000000002, 0.0], [2725.6700000000001, 22.5], [2299.73, 45.0], [1830.8299999999999, 67.5], [1309.5, 90.0]]],
['LFJ1', [[3376.4699999999998, 0.0], [2495.5999999999999, 22.5], [1755.72, 45.0], [921.65999999999997, 67.5], [443.10000000000002, 90.0]]],
['LFJ3', [[1732.48, 0.0], [2103.4299999999998, 22.5], [2392.7600000000002, 45.0], [2718.6599999999999, 67.5], [2923.1399999999999, 90.0]]],
['THJ1', [[2276.52, 0.0], [1871.78, 22.5], [1466.6199999999999, 45.0], [1160.03, 67.5], [914.54999999999995, 90.0]]],
['THJ2', [[2244.04, -30.0], [2083.3800000000001, -15.0], [1925.46, 0.0], [1750.72, 15.0], [1621.8599999999999, 30.0]]],
['LFJ5', [[415.29000000000002, 0.0], [1375.6199999999999, 22.5], [1838.1099999999999, 45.0], [2220.71, 67.5], [2808.4299999999998, 90.0]]]
]
